tests/suite/system_suite_test.go

```diff
@@ -173,6 +174,11 @@ func setup(cfg setupConfig, extraInstallArgs ...string) {
 		}
 		installCfg.ImageTag = *imageTag
 		installCfg.ImagePullPolicy = *imagePullPolicy
+	} else {
+		if version == "edge" {
+			chartVersion = "0.0.0-edge"
+			installCfg.ChartVersion = chartVersion
+		}
 	}
```

Comment:
does the variable `chartVersion` get set in the pipeline?

Reply:
No it only gets set here when the version is `edge`